_docs/setup/kubernetes/multicluster-install.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Istio Multicluster
+overview: Install Istio with multicluster support.
+
+order: 65
+
+layout: docs
+type: markdown
+---
+
+{% include home.html %}
+
+Instructions for the installation of Istio multicluster.
+
+## Prerequisites
+
+* Two or more Kubernetes clusters.
+
+* The usage of a RFC1918 network, VPN, or other more advanced network techniques
+to meet the following requirements:
+
+    * Individual cluster Pod CIDR ranges and service CIDR ranges must be unique
+across the multicluster environment and may not overlap.
+
+    * All nodes' pod CIDR in every cluster must be routeable to every other nodes'
+pod CIDR.
+
+    * All Kubernetes control plane API servers must be routeable to each other.
+
+* Helm **2.7.2 or newer**.  The use of Tiller is optional.
+
+* If using Tiller, the ability to modify RBAC rules required to install Helm
+or alternatively Helm should already installed.
+
+* Kubernetes **1.7.3 or newer**.
+
+* Istio secret `istio.default` replicated to all remote Kubernetes clusters
+from the Istio control plane per application namespace.
+
+* The local cluster and all remote Kubernetes cluster credentials (typically
+stored in `$HOME/.kube/config`) copied to
+`$HOME/multicluster/${UNIQUE_IDENTIFIER}.kube.conf`.
+
+* A [deployed Istio control plane]({{home}}/docs/setup/kubernetes/quick-start.html)
+on **one** Kubernetes cluster.  Note that Istio must be [deployed with Helm]({{home}}/docs/setup/kubernetes/helm-install.html) using the `--set global.multicluster.enabled=true` flag.  This is currently mandatory.
+
+* Currently only [manual sidecar injection]({{home}}/docs/setup/kubernetes/sidecar-injection.html)
+has been validated with multicluster.
+
+## Install a multicluster configmap on the Istio control plane
+
+**Important**: Pilot will not start until the configmap in these instructions
```

Comment:
Remove the **Important**. and replace with ">" to create a block note. If this is really really important, you can also add the exclamation mark imagery (search for exclamation in the docs for example uses)

Reply:
done.